### PR TITLE
bitmessageqt address_dialogs quality fixes

### PR DESCRIPTION
**Pull request by coolguy-cell**
flake8 fixes for address_dialogs
pycodestyle fixes for address_dialogs
pylint fixes for address_dialogs

### CODE REVIEW

File: src/bitmessageqt/address_dialogs.py [hunk 1]
```diff
@@ -90,13 +105,18 @@ def accept(self):
         if self.valid:
             self.data = (
                 addBMIfNotPresent(str(self.lineEditAddress.text())),
-                str(self.lineEditLabel.text().toUtf8())
+                str(self.lineEditLabel.text().toUtf8()),
```
Comment (PeterSurda) [inline src/bitmessageqt/address_dialogs.py:108]: why?

File: src/bitmessageqt/address_dialogs.py [hunk 2]
```diff
@@ -138,62 +158,75 @@ def accept(self):
...
+            if self.lineEditPassphrase.text() != self.lineEditPassphraseAgain.text():
                 QtGui.QMessageBox.about(
-                    self, _translate("MainWindow", "Passphrase mismatch"),
+                    self,
+                    _translate("MainWindow", "Passphrase mismatch"),
                     _translate(
                         "MainWindow",
-                        "The passphrase you entered twice doesn\'t"
-                        " match. Try again.")
+                        "The passphrase you entered twice doesn't" " match. Try again.",
```
Comment (PeterSurda) [inline src/bitmessageqt/address_dialogs.py:181]: no `" "` in the middle

File: src/bitmessageqt/address_dialogs.py [hunk 3]
```diff
@@ -138,62 +158,75 @@ def accept(self):
...
         if addressVersion <= 3:
-            self.checkBoxDisplayMessagesAlreadyInInventory.setText(_translate(
-                "MainWindow",
-                "Address is an old type. We cannot display its past"
-                " broadcasts."
-            ))
+            self.checkBoxDisplayMessagesAlreadyInInventory.setText(
+                _translate(
+                    "MainWindow",
+                    "Address is an old type. We cannot display its past" " broadcasts.",
```
Comment (PeterSurda) [inline src/bitmessageqt/address_dialogs.py:220]: `" "`

**Comment by coolguy-cell** [comment]
sure

File: src/bitmessageqt/address_dialogs.py [hunk 4]
```diff
@@ -232,38 +268,35 @@ class SpecialAddressBehaviorDialog(QtGui.QDialog):
...
-                self.lineEditMailingListName.setText(_translate(
-                    "SpecialAddressBehaviorDialog",
-                    "This is a chan address. You cannot use it as a"
-                    " pseudo-mailing list."
-                ))
+                self.lineEditMailingListName.setText(
+                    _translate(
+                        "SpecialAddressBehaviorDialog",
+                        "This is a chan address. You cannot use it as a"
+                        "pseudo-mailing list."
```
Comment (PeterSurda) [inline src/bitmessageqt/address_dialogs.py:287]: missing space

File: src/bitmessageqt/address_dialogs.py [hunk 5]
```diff
@@ -296,20 +331,24 @@ def accept(self):
...
-                " please try a new one."
-            ))
+            self.setWindowTitle(
+                _translate("EmailGatewayDialog", "Registration failed:")
+            )
+            self.label.setText(
+                _translate(
+                    "EmailGatewayDialog",
+                    "The requested email address is not available,"
+                    "please try a new one."
```
Comment (PeterSurda) [inline src/bitmessageqt/address_dialogs.py:349]: missing space

File: src/bitmessageqt/bitmessage_icons_rc.py [hunk 6]
```diff
@@ -1,12 +1,14 @@
 # -*- coding: utf-8 -*-
-
+"""
```
Comment (PeterSurda) [inline src/bitmessageqt/bitmessage_icons_rc.py:2]: find a proper module docstring

**Comment by g1itch** [comment]
Can you please tell me why this formatting is better than my? Because I spent many hours trying to save horizontal space and make the code more readable without PEP8 violation. This looks like pasta for me and if it will be merged I'm going to revert most of this changes during the nest rebase of my qt5-wip branch.

**Comment by PeterSurda** [comment]
@g1itch you're right, this is my bad, I wasn't paying attention to the results of the automated checks.

**Pull request closed without merging**